Server/src/models/entities/Units.ts

```diff
@@ -1,25 +1,55 @@
-import { Entity, Column, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn } from "typeorm";
+import { BaseEntity, Column, CreateDateColumn, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn, UpdateDateColumn } from "typeorm";
+import { IUnitModel } from "../interfaces/IDimension";
 
+import { Dimension } from "./Dimension";
 
 /**
  * The entity annotation indicates that a table is being created
  * This entity handles units (ex. GPa, km/s) of data points
  */
 @Entity()
-export class Units {
-
+export class Units extends BaseEntity {
     @PrimaryGeneratedColumn()
     id: number
 
-    @Column({ length: 50 })
+    @Column({ length: 10 })

```

Comment:
why was this decresed??? a name may possibly jump over 10 chars

Reply:
This was not decreased, but the old 'name' column was removed and the existing 'units' column was changed to 'name' for better clarity.